bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,913 @@
+
+	<thing-type id="t19">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T19</label>
+		<description>Single Color LED Strip</description>
+
+		<channels>
+			<channel id="onOff" typeId="onoff"/>
+			<channel id="rollerBrightness" typeId="rollerBrightness"/>
+			<channel id="dimmerBrightness" typeId="dimmerBrightness"/>
+			<channel id="sleep" typeId="sleep"/>
+			<channel id="lastStatusStored" typeId="lastStatusStored"/>
+			<channel id="healthy" typeId="healthy"/>
+		</channels>
+
+		<config-description>
+			<parameter name="sleep" type="integer" min="1" max="1000">
+				<label>Sleep</label>
+				<description>Set sleep timer in cycles</description>
+				<required>false</required>
+				<default>5</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="t1a">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T1A</label>
+		<description>Digital input pass through</description>
+
+		<channels>
+			<channel id="one" typeId="stateOpenClose">
+				<label>1</label>
+			</channel>
+			<channel id="two" typeId="stateOpenClose">
+				<label>2</label>
+			</channel>
+			<channel id="three" typeId="stateOpenClose">
+				<label>3</label>
+			</channel>
+			<channel id="four" typeId="stateOpenClose">
+				<label>4</label>
+			</channel>
+			<channel id="five" typeId="stateOpenClose">
+				<label>5</label>
+			</channel>
+			<channel id="six" typeId="stateOpenClose">
+				<label>6</label>
+			</channel>
+			<channel id="seven" typeId="stateOpenClose">
+				<label>7</label>
+			</channel>
+			<channel id="eight" typeId="stateOpenClose">
+				<label>8</label>
+			</channel>
+			<channel id="lastStatusStored" typeId="lastStatusStored"/>
+			<channel id="healthy" typeId="healthy"/>
+		</channels>
+	</thing-type>
+
+	<thing-type id="t21">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T21</label>
+		<description>Motorized devices with limit switches</description>
+		<channels>
+			<channel id="rollershutter" typeId="rollershutter"/>
+			<channel id="rollershutterstate" typeId="rollershutterstate"/>
+			<channel id="lastStatusStored" typeId="lastStatusStored"/>
+			<channel id="healthy" typeId="healthy"/>
+		</channels>
+		<config-description>
+			<parameter name="secureSend" type="boolean">
+				<label>Secure Send</label>
+				<description></description>
+				<required>false</required>
+				<default>true</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="t22">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T22</label>
+		<description>Motorized devices with limit switches and middle position</description>
+		<channels>
+			<channel id="rollershutter" typeId="rollershutter"/>
+			<channel id="rollershutterstate" typeId="rollershutterstate"/>
```

Comment:
Please use lowerCamelCase for all of the channel IDs.

Reply:
fixed